moler/cmd/unix/sudo.py

```diff
@@ -233,7 +243,19 @@ def _parse_password(self, line):
         """
         if re.search(self._get_password_regex(), line):
             if not self._sent_password:
-                self.connection.sendline(self.password, encrypt=self.encrypt_password)
+                try:
+                    pwd = self.password.pop(0)
```

Comment:
hmm, maybe different name would be more readable. At first glance I thought that you send `pwd` command :D

Reply:
Done. `current_password`.